apps/frontend/src/app/Database/DataEntry.ts

```diff
@@ -81,8 +97,8 @@ export class DataEntry<
     this.database.storage.set(this.key, this.deCache(this.data))
   }
 
-  trigger(reason: TriggerString, object?: any) {
-    this.listeners.forEach((cb) => cb(reason, object))
+  trigger(reason: TriggerString, object?: unknown) {
```

Comment:
Why can't this be typed as `CacheValue`

Reply:
In invalid cases we are not passing in a "valid" `CacheValue`

Reply:
Doesn't that make this an incorrect type assertion?

Reply:
yes. need to have an overhaul of the event trigger system to have "correct types", which isnt the scope of this PR.